extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -346,7 +346,7 @@ export class SchemaCompareMainWindow {
 		this.resetButtons(ResetButtonState.afterCompareComplete);
 
 		if (this.comparisonResult.differences.length > 0) {
-			this.flexModel.addItem(this.splitView, { CSSStyles: { 'overflow': 'hidden' } });
```

Comment:
How does this look for very long strings?

Reply:
@ellbosch can you please add a snapshot of how SC looks before/after the change

Reply:
Yes please add gifs

Reply:
**Old functionality:**
![old](https://user-images.githubusercontent.com/4683017/64461299-a9ea3c80-d0b1-11e9-87e8-88ee6173c6ce.gif)

**New functionality:**
![new](https://user-images.githubusercontent.com/4683017/64461300-a9ea3c80-d0b1-11e9-90a6-2e3d6c769915.gif)